compiler/noirc_driver/src/abi_gen.rs

```diff
@@ -197,9 +198,7 @@ pub(super) fn value_from_hir_expression(context: &Context, expression: HirExpres
             },
             HirLiteral::Bool(value) => AbiValue::Boolean { value },
             HirLiteral::Str(value) => AbiValue::String { value },
-            HirLiteral::Integer(field, sign) => {
-                AbiValue::Integer { value: field.to_string(), sign }
-            }
+            HirLiteral::Integer(field, sign) => AbiValue::Integer { value: field.to_hex(), sign },
```

Comment:
@TomAFrench I just tried using this fix and it causes an issue in the transpiler [here](https://github.com/AztecProtocol/aztec-packages/blob/44853d17b66e212d3ce200242979ef0c6ee73970/avm-transpiler/src/main.rs#L45) because it doesn't expect hex values (they don't seem to be natively support by the json format see [here](https://stackoverflow.com/a/52671914)).

I think it would be preferable to stick to the non-hex format to have the artifacts standard JSON compatible.

One way to do it is what [Grego did yesterday](https://github.com/AztecProtocol/aztec-packages/pull/7066/commits/cfc52339ea12db3f81bb693a34ffe3d69f194181) (see the last change in the diff) and convert it to u128 first.

Reply:
Tried using the non-hex format as well and getting the same error. So it's probably some other issue.

Reply:
Clean bootstrap resolved this. Ignore me